sympy/crypto/crypto.py

```diff
@@ -1266,44 +1266,55 @@ def rsa_private_key(p, q, e):
 
 def encipher_rsa(i, key):
     """
-    Return encryption of ``i`` by computing `i^e` (mod `n`),
-    where ``key`` is the public key `(n, e)`.
+    Return encryption of "i" by computing 'i^e' (mod 'n'),
+    where "key" is the key '(n, e)'. "key" may be a public
+    key or a private key.
 
     Examples
     ========
-
+    *Public Key Encryption*
     >>> from sympy.crypto.crypto import encipher_rsa, rsa_public_key
     >>> p, q, e = 3, 5, 7
     >>> puk = rsa_public_key(p, q, e)
     >>> msg = 12
     >>> encipher_rsa(msg, puk)
     3
-
+    *Private Key Encryption*
+    >>> from sympy.crypto.crypto import encipher_rsa, rsa_private_key
+    >>> p, q, e = 3, 5, 7
+    >>> prk = rsa_private_key(p, q, e)
+    >>> msg = 12
+    >>> encipher_rsa(msg, prk)
+    3
     """
     n, e = key
     return pow(i, e, n)
 
 
 def decipher_rsa(i, key):
     """
-    Return decyption of ``i`` by computing `i^d` (mod `n`),
-    where ``key`` is the private key `(n, d)`.
+    Return decyption of "i" by computing 'i^d' (mod 'n'),
+    where "key" is the key '(n, d)'. "key" may be a public
+    key or a private key.
 
     Examples
     ========
-
     >>> from sympy.crypto.crypto import decipher_rsa, rsa_private_key
     >>> p, q, e = 3, 5, 7
     >>> prk = rsa_private_key(p, q, e)
-    >>> msg = 3
-    >>> decipher_rsa(msg, prk)
+    >>> puk = rsa_public_key(p, q, e)
+    >>> msg = 12
+    >>> new_msg = encipher_rsa(msg, prk)
```

Comment:
The test may be failing because some functions like these are not imported.

Besides, I think the examples can be formatted as `encipher_rsa`
There can be two sections like `Public Key Decryption` and `Private Key Decryption`,
and add corresponding processes, matching examples above